From: Simon McVittie <smcv@debian.org>
Date: Fri, 29 Jan 2016 13:15:50 +0100
Subject: Don't run tests under gtester in addition to "make check"

We don't get any additional test coverage by doing this;
it makes sense to put a given test in either TESTS or TEST_PROGS,
but not both.

I've chosen TESTS because Automake's test harness can cope with tests
that exit 77 and recognise that as "all test-cases were skipped",
whereas gtester currently can't.
---
 tests/Makefile.am.inc | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tests/Makefile.am.inc b/tests/Makefile.am.inc
index 2475852..023d9a0 100644
--- a/tests/Makefile.am.inc
+++ b/tests/Makefile.am.inc
@@ -1,4 +1,3 @@
-TEST_PROGS += testdb test-doc-portal
 testdb_CFLAGS = $(BASE_CFLAGS) -DDB_DIR=\"$(abs_srcdir)/tests/dbs\"
 testdb_LDADD = \
              $(BASE_LIBS) \
@@ -28,7 +27,7 @@ tests/services/org.freedesktop.XdgApp.service: session-helper/org.freedesktop.Xd
 
 test_doc_portal_DEPENDENCIES = tests/services/org.freedesktop.portal.Documents.service  tests/services/org.freedesktop.XdgApp.service
 
-check_PROGRAMS = $(TEST_PROGS)
+check_PROGRAMS = $(TESTS)
 
 TESTS=testdb test-doc-portal
 
